modules/ROOT/partials/configuration/tinycomments_lookup.adoc

```diff
@@ -48,10 +48,10 @@ For example:
 
 [source,js]
 ----
-function lookup_comment({ conversationUid }, done, fail) {
-  let lookup = async function () {
+const lookup_comment = ({ conversationUid }, done, fail) => {
+  let lookup = async () => {
```

Comment:
Should these be `let` variables or `const` variables in this function?

- [ ] Swap to using `const` if these aren't mutable

Reply:
Done